Tests/InstrumentationTests/Tracing/TracedLock.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import Baggage
+import Instrumentation
+
+final class TracedLock {
```

Comment:
note to self, add docs;

This is a fun example how one can do a lock that traces how long it keeps the critical section for each user.

Reply:
Thanks for adding this example. Looks good 👍